src/styles/utils/variables.scss

```diff
@@ -0,0 +1,24 @@
+// header var
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)